kubetest/gke.go

```diff
@@ -287,23 +290,27 @@ func (g *gkeDeployer) IsUp() error {
 //
 // TODO(zmerlynn): This whole path is really gross, but this seemed
 // the least gross hack to get this done.
+//
+// TODO(shyamjvs): Make this work with multizonal clusters.
```

Comment:
and regional

Reply:
Done. Btw - isn't regional a subset of multi-zonal (where all zones are from same region)?

Reply:
multi-zonal is always within a single region.

It doesn't have to be a subset - you can have a regional GKE cluster, where all nodes will be in a single zone.